### ISSUE TITLE
[MacOS] Meshes and Actors with applied fbx. assets have graphical artifacts on MacBook Air machine with M1 chip

### ISSUE THREAD

**Issue by LB-MateuszSzweda** [o3de/o3de#3413]
**Describe the bug**
It has been observed that all of fbx. assets assigned to entities with mesh or actor component will appear graphically distorted on a specific MacBook Air machine.

The machine is a MacBook Air (M1, 2020) with macOS Big Sur version 11.5.1 and Xcode version 12.5.1 (12E507) .

For further information please refer to the attached video file.

https://user-images.githubusercontent.com/86953456/130620833-e11953b3-1ba5-4714-bf40-1f053e23a0d4.mov

**Prerequisites**
* Asset Processor and Editor has been built.
* Assets were processed in Asset Processor.
* AutomatedTesting was set as a project to launch.

**To Reproduce**
1. Launch the Editor.
2. Create a new level.
3. Add a new entity with mesh component to the level.
4. Assign the mesh component of the entity with any fbx. asset.
5. Move the editor camera close to the created entity.

**Expected behavior**
Meshes and Actors with assigned assets appear normal and are free of artifacts.

**Desktop/Device (please complete the following information):**
- Device: Mac
- OS: MacOS
- Version: Big Sur 11.5.1
- CPU: M1
- GPU: M1
- Memory: 8GB

**Comment by wintermute-motherbrain** [comment]
This issue has been resolved in development.  Can you please verify this is still happening?  Note* you may need to manually delete the shader cache folder to fix the issue.

**Comment by LB-MateuszSzweda** [comment]
Today verifying the fix was blocked by https://github.com/o3de/o3de/issues/3888 preventing any .fbx assets to be added to actor and meshes due to Asset Processor failure but I will verify it as soon as it is resolved.

**Comment by LB-MateuszSzweda** [comment]
Unfortunately after #3888 was resolved and fbx. assets were again usable the graphical issue with meshes and actors on M1 machine were still occuring. It was tested on development build with _daf901e_ (09-03) as the latest commit.

I am attaching a screenshot of the issue from today.

<img width="1440" alt="09-03 M1 meshes and actors" src="https://user-images.githubusercontent.com/86953456/132014707-64227ba0-7d27-4fee-8a63-2e346fd68a94.png">

**Comment by LB-MateuszSzweda** [comment]
I was able to confirm that on Development (ba99568) the issue no longer occurs.

Meshes and Actors provided with _.fbx_ assets including the ones from repro steps were not displaying graphical artifacts on M1 Macbook Air in the Editor and the Launcher. 

Therefore the issue can be treated as fixed and closed.

**Comment by godpiao** [comment]
> I was able to confirm that on Development ([ba99568](https://github.com/o3de/o3de/commit/ba99568a4f92f239151a0b78713ae9bbe94c5ce3)) the issue no longer occurs.
> 
> Meshes and Actors provided with _.fbx_ assets including the ones from repro steps were not displaying graphical artifacts on M1 Macbook Air in the Editor and the Launcher.
> 
> Therefore the issue can be treated as fixed and closed.

you can run o3de in m1?

**Comment by LB-MateuszSzweda** [comment]
@godpiao Yes. o3de was correctly running on OSX machine with m1 chipset on march and today I was able to verify that it also builds and launches correctly on M1 chipset OSX machine with macOS Monterey 12.4 with Development (146b2c8) build.

The only difference between OSX machine without M1 chipset is adding additional "-DCMAKE_OSX_ARCHITECTURES=x86_64" line to configuring command.

The issue described in the ticket was also not observed today.
